src/__tests__/__actions__/authActions.test.js

```diff
@@ -65,7 +65,7 @@ describe("Login action creators", () => {
         }
       ];
 
-      moxios.stubRequest(`${process.env.API_BASE_URL}/users/login`, {
+      moxios.stubRequest(`${process.env.API_URL}/users/login`, {
```

Comment:
@niomwungeri-fabrice Can't you destruct API_URL?

Reply:
Hi @oesukam!
Those aren't my changes but I can change them.
Thanks